app/controllers/application_controller.rb

```diff
@@ -1,9 +1,20 @@
+require "application_responder"
```

Comment:
What do you think if me move responder into `app/responders/application_responder.rb`. In this case we can remove this line.